src/aap_eda/services/project/scm.py

```diff
@@ -257,8 +258,18 @@ class GitAnsibleRunnerExecutor:
     def __call__(
         self,
         extra_vars: dict,
+        proxy: str,
     ):
         with tempfile.TemporaryDirectory(prefix="EDA_RUNNER") as data_dir:
+            if proxy:
```

Comment:
@bzwei I think according to this doc https://ansible.readthedocs.io/projects/runner/en/latest/ansible_runner/#ansible_runner.interface.run we don't have to create a directory we can directly pass in via enviers
```
envvars ([dict](https://docs.python.org/3/library/stdtypes.html#dict)) – Environment variables to be used when running Ansible. Environment variables will also be read from env/envvars in private_data_dir
```